src/main/app/forms/validation/validators/minNumberOfPopulatedRows.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  registerDecorator,
+  ValidationArguments,
+  ValidationOptions,
+  ValidatorConstraint,
+  ValidatorConstraintInterface
+} from 'class-validator'
+
+import { MultiRowForm } from 'forms/models/multiRowForm'
+import { MultiRowFormItem } from 'forms/models/multiRowFormItem'
+
+@ValidatorConstraint()
+export class MinNumberOfPopulatedRowsConstraint implements ValidatorConstraintInterface {
```

Comment:
`AtLeastOnePopulatedRow`?
The name here looks like it is configurable, but it isn't

Reply:
yeah, you're right

Reply:
done